src/libPMacc/include/particles/ParticlesBase.kernel

```diff
@@ -689,97 +689,200 @@ struct KernelDeleteParticles
     }
 };
 
+/** copy particle from the border to a exchange buffer
+ *
+ * @tparam T_numWorkers number of workers
+ */
+template< uint32_t T_numWorkers >
 struct KernelBashParticles
```

Comment:
question: maybe this kernel is a bit ill-named.
why not rename it to `kernelCopyBorderToExchange` ?